pkg/epoch/epoch_test.go

```diff
@@ -30,7 +31,7 @@ func rightAfter(t1, t2 time.Time) bool {
 		// Low timer resolution on Windows
 		return (t2.After(t1) && t2.Before(t1.Add(100*time.Millisecond))) || t2.Equal(t1)
 	}
-	return t2.After(t1) && t2.Before(t1.Add(10*time.Millisecond))
+	return (t2.After(t1) && t2.Before(t1.Add(10*time.Millisecond))) || t2.Equal(t1)
 }
```

Comment:
```
threshold := 10 * time.Millisecond
if runtime.GOOS == "windows"{
  // Low timer resolution on Windows
  threashold *= 10
}
return (t2.After(t1) && t2.Before(t1.Add(threshold))) || t2.Equal(t1)
```

Reply:
Ah, LOL I was thinking about that after pushing; yes, let me do that; I guess I can also return early on `t2.Equal()`

Reply:
Updated, PTAL 👍

Reply:
oh let me change thresHold to threshold (lowercase) my bad 😂 

(done!)